src/librustdoc/passes/strip_hidden.rs

```diff
@@ -40,7 +40,7 @@ impl<'a> DocFolder for Stripper<'a> {
             debug!("strip_hidden: stripping {:?} {:?}", i.type_(), i.name);
             // use a dedicated hidden item for given item type if any
             match *i.kind {
-                clean::StructFieldItem(..) | clean::ModuleItem(..) => {
+                clean::StructFieldItem(..) | clean::ModuleItem(..) | clean::VariantItem(..) => {
```

Comment:
Without this, the hidden variants are removed, preventing us to correctly use `has_stripped_entries`.